specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/examples/ApiManagementListTenantSettings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "parameters": {
+    "serviceName": "apimService1",
+    "resourceGroupName": "rg1",
+    "api-version": "2020-06-01-preview",
+    "subscriptionId": "subid"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [],
```

Comment:
>    "value": [], [](start = 4, length = 16)

Provide actual example to understand how will it look

Reply:
We have this scenario: we have just one entity, "accessInformation" which contains some properties customer needs to generate his passwords.  There are no collection of such objects, it's just one entity. We historically created endpoint /tenant/access with the GET/PUT methods and no any collections. This endpoint meant "some entity related to not any collection but to entire tenant only." To comply with requirements that "anything should be a collection" we introduced /tenant endpoint in the current PR, which actually return empty array since we don't have any collection here. 
Please advise what is the proper way to implement GET/PUT endpoint for the entity without collection.